README.md

```diff
@@ -1,8 +1,24 @@
 # Octokit.swift
 
 [![Build Status](https://travis-ci.org/nerdishbynature/octokit.swift.svg?branch=master)](https://travis-ci.org/nerdishbynature/octokit.swift)
+[![CocoaPods](https://img.shields.io/cocoapods/v/OctoKit.swift.svg)](https://cocoapods.org/pods/OctoKit.swift)
 [![codecov.io](https://codecov.io/github/nerdishbynature/octokit.swift/coverage.svg?branch=master)](https://codecov.io/github/nerdishbynature/octokit.swift?branch=master)
 
+## Installation
+
+- **Using [Swift Package Manager](https://swift.org/package-manager)**:
+
+    ```swift
+    import PackageDescription
+
+    let package = Package(
+      name: "MyAwesomeApp",
+      dependencies: [
+        .Package(url: "https://github.com/nerdishbynature/octokit.swift", from: "0.8.0"),
+      ]
+    )
+    ```
```

Comment:
Using both indentation and backticks is redundant. I suggest removing the indentation in favor of the swift code block, plus it will be a little easier to keep this updated.